Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -2631,6 +2654,149 @@ script:
+      description: Status of application.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.UpdatedAt
+      description: Application updated at.
+      type: string
+    - contextPath: SentinelOne.InstalledApps.ApplicationInstanceName
+      description: Application instance name.
+      type: string
+  - arguments:
+    - name: account_ids
+      description: A comma-separated list of account IDs.
+      isArray: true
+    - name: role_ids
+      description: A comma-separated list of rbac roles to filter by.
+      isArray: true
+    - name: ids
+      description: A comma-separated list of service user IDs to filter by.
+      isArray: true
+    - name: site_ids
+      description: A comma-separated list of site IDs.
+      isArray: true
+    description: Returns all service users that match the specified filter values.
+    name: sentinelone-get-service-users
+    outputs:
+    - contextPath: SentinelOne.ServiceUsers.ID
+      description: The service user ID.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.ApiTokenCreatedAt
+      description: Api token created at.
+      type: date
+    - contextPath: SentinelOne.ServiceUsers.ApiTokenExpiresAt
+      description: Api token expires at.
+      type: date
+    - contextPath: SentinelOne.ServiceUsers.CreatedAt
+      description: Service user created at.
+      type: date
+    - contextPath: SentinelOne.ServiceUsers.CreatedById
+      description: The service user created by Id.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.CreatedByName
+      description: The service user created by name.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.Description
+      description: Service user description.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.LastActivation
+      description: Last activation date.
+      type: date
+    - contextPath: SentinelOne.ServiceUsers.Name
+      description: Service user name.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.Scope
+      description: Service user scope.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.UpdatedAt
+      description: Service user updated at.
+      type: date
+    - contextPath: SentinelOne.ServiceUsers.UpdatedById
+      description: Service user updated by Id.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.UpdatedByName
+      description: Service user updated by name.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.ScopeRolesRoleId
+      description: Scope roles role Id.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.ScopeRolesRoleName
+      description: Scope roles role name.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.ScopeRolesAccountName
+      description: Scope roles account name.
+      type: string
+    - contextPath: SentinelOne.ServiceUsers.ScopeRolesId
+      description: Scope roles Id.
```

Comment:
```suggestion
      description: Service user updated by ID.
      type: string
    - contextPath: SentinelOne.ServiceUsers.UpdatedByName
      description: Service user updated by name.
      type: string
    - contextPath: SentinelOne.ServiceUsers.ScopeRolesRoleId
      description: Scope roles role ID.
      type: string
    - contextPath: SentinelOne.ServiceUsers.ScopeRolesRoleName
      description: Scope roles role name.
      type: string
    - contextPath: SentinelOne.ServiceUsers.ScopeRolesAccountName
      description: Scope roles account name.
      type: string
    - contextPath: SentinelOne.ServiceUsers.ScopeRolesId
      description: Scope roles ID.
```

Reply:
It will not change because it is not related to the above contribution but to the commit from master.